test/pool/FeesEscrow.test.js

```diff
@@ -0,0 +1,117 @@
+const { contracts, contractSettings } = require('../../deployments/settings');
+const {setupOracleAccounts, impersonateAccount} = require("../utils");
+const {upgradeContracts} = require("../../deployments");
+const {send, ether, BN} = require("@openzeppelin/test-helpers");
+const {ethers} = require("hardhat");
+const {expect} = require("chai");
+
+const Pool = artifacts.require('Pool');
+const StakedEthToken = artifacts.require('StakedEthToken');
+const RewardEthToken = artifacts.require('RewardEthToken');
+const PoolValidators = artifacts.require('PoolValidators');
+const Oracles = artifacts.require('Oracles');
+
+let pool;
+let feesEscrow;
+let stakedEthToken;
+let rewardEthToken;
+let validators;
+let oracles;
+let oracleAccounts;
+const admin = contractSettings.admin;
+
+async function upgradeRewardEthToken() {
```

Comment:
This should be called from deployments/index.js so that other tests would run on top of upgraded contracts